src-tauri/src/main.rs

```diff
@@ -1,15 +1,32 @@
 // Prevents additional console window on Windows in release, DO NOT REMOVE!!
 #![cfg_attr(not(debug_assertions), windows_subsystem = "windows")]
 
-// Learn more about Tauri commands at https://tauri.app/v1/guides/features/command
-#[tauri::command]
-fn greet(name: &str) -> String {
-    format!("Hello, {}! You've been greeted from Rust!", name)
-}
+use tauri::Manager;
+use utils::setup::{setup_app_state, setup_store};
+
+mod commands;
+mod error;
+mod operations;
+mod store;
+mod types;
+mod utils;
 
-fn main() {
+fn main() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
yeah, eyre would be the way to go

```suggestion
fn main() -> eyre::Result<()> {
```